sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1841,15 +2118,29 @@ class ReadFromBigQuery(PTransform):
     on GCS, and then reads from each produced file. File format is Avro by
     default.
 
+  .. warning::
```

Comment:
should we also update document description here?

https://github.com/apache/beam/pull/15185/files#diff-0c77ebbb0792b80a76f4ec1f2b89b9d319f5e33f1ee0857ed6fe414f75c48cbbR2117

Reply:
I'm not sure what you would like to see changed in the document description. Were you pointing to another location instead?

Reply:
Sorry for confusion, I've screenshotted the snippet of docs. I assume that RFBQ does not use only export job in BQ after this PR?

![Screenshot from 2021-08-16 09-41-00](https://user-images.githubusercontent.com/2764630/129599119-d6639865-1cdf-4afd-8eb0-0ff6452a6d47.png)

Reply:
It should still use export jobs when the mode is set to "EXPORT" (the default)

Reply:
(yeah, doc should be updated to clarify that and also describe modes).

Reply:
Done